app/services/idv/proofer.rb

```diff
@@ -31,10 +29,13 @@ def get_vendor(stage)
       end
 
       def configure
-        self.configuration ||= Configuration.new
         yield(configuration)
       end
 
+      def configuration
```

Comment:
Allows `configuration` to be set to default on use in case `init` is not called.